documentation/CONFIG.md

```diff
@@ -628,7 +629,33 @@ service:
       - ./.env
 ```
 
-You also can add these variables to the call of  the `docker-compose` command. For example like this `docker-compose up -e PORT_APP=8080`. 
+You also can add these variables to the call of  the `docker-compose` command. For example like this `docker-compose up -e PORT_APP=8080`.
+
+#### GPS
+
+If present, OpenDataCam can obtain the current position via GPS and persist it along other counter data. To enable GPS add the following section to your `config.json`
```

Comment:
Maybe you could add a flag enabled that default to false instead of enabling it if present in the `config.json` .. this would make the discovery of this feature easier..

Could you please also add some quick words giving an example of GPS device that can work and how do you setup it ? I understood if you need to plug it via usb ?

Also could you make it explicit that this is only usefull if you use a moving camera ?

```
"GPS": {
  "enabled": false
  "port": 2947,
  "hostname": "localhost",
  "signalLossTimeoutSeconds": 60,
  "csvExportOpenStreeetMapsUrl": true
}
```

Reply:
Let me know if this is better.